File: python/awqat/prayer.py
"""PrayerTime (Namaz Vakitleri) endpoint'leri. İstemciyi ilk argüman alır."""


def daily(c, city_id):
    return c.get_json(f"/api/PrayerTime/Daily/{city_id}")


def weekly(c, city_id):
    return c.get_json(f"/api/PrayerTime/Weekly/{city_id}")


def monthly(c, city_id):
    return c.get_json(f"/api/PrayerTime/Monthly/{city_id}")


def ramadan(c, city_id):
    return c.get_json(f"/api/PrayerTime/Ramadan/{city_id}")


def eid(c, city_id):
    return c.get_json(f"/api/PrayerTime/Eid/{city_id}")

File: php/awqat/Prayer.php
<?php

declare(strict_types=1);

namespace Awqat;

/** PrayerTime (Namaz Vakitleri) endpoint'leri. İstemciyi ilk argüman alır. */
class Prayer
{
    public static function daily(Client $c, int $cityId)
    {
        return $c->getJson("/api/PrayerTime/Daily/$cityId");
    }

    public static function weekly(Client $c, int $cityId)
    {
        return $c->getJson("/api/PrayerTime/Weekly/$cityId");
    }

    public static function monthly(Client $c, int $cityId)
    {
        return $c->getJson("/api/PrayerTime/Monthly/$cityId");
    }

    public static function ramadan(Client $c, int $cityId)
    {
        return $c->getJson("/api/PrayerTime/Ramadan/$cityId");
    }

    public static function eid(Client $c, int $cityId)
    {
        return $c->getJson("/api/PrayerTime/Eid/$cityId");
    }
}
